plugins/modules/azure_rm_aks_info.py

```diff
@@ -44,6 +44,11 @@
 
 author:
     - Yuwei Zhou (@yuwzho)
+
+deprecated:
+  removed_in: '2.0.0'
```

Comment:
Just reading https://docs.ansible.com/ansible/devel/dev_guide/module_lifecycle.html it seems to say that we can only add removed_in after 4 cycles.  Wouldn't that mean ansible 9.0?

Reply:
@pabelanger good catch.

It would be accurate to say we are making an exception here and short-circuiting the normal process because it turns out that this collection is an unmaintained duplicate of azure.azcollection and there is no interest in keeping it around that long.

Reply:
@dmsimard Yah, agree. We maybe should call this out in the documentation explaining why we are doing it here, so other also know.  Given there is another (more adopted) collection we seem to be in the clear.  However, as a passer by to the issue it wasn't clear to me why we bypassed it.